### PR TITLE
Cal checkerboard object coords update

### PR DESCRIPTION
**Pull request by gerth2**
Adds additional checkerboad calibration test from images taken from a Pi3B and Raspberry Pi Camera V2 at 320x240.

Added logic to test to produce a series of "undistorted" images for visually debugging the results of the calibration.

Updated generation of checkerboard real-world object coordinates to produce a square pattern, not a parallelogram pattern.

Added target support for the 2016 High Goal target.

### CODE REVIEW

**Comment by gerth2** [comment]
Per discord chat - I don't think we're as-of-yet convinced _why_ this was producing an error factor of ~10x in my setup, but it seems to have corrected it? Probably want someone else to confirm on gloworm.

File: photon-client/src/views/PipelineViews/PnPTab.vue [hunk 1]
```diff
@@ -60,7 +60,7 @@
         },
         data() {
             return {
-                targetList: ['2020 High Goal Outer', '2020 High Goal Inner', '2019 Dual Target', 'Power Cell (7in)'],
+                targetList: ['2020 High Goal Outer', '2020 High Goal Inner', '2019 Dual Target', 'Power Cell (7in)', '2016 High Goal'], //Keep in sync with TargetModel.java
```
Comment (mcm001) [inline photon-client/src/views/PipelineViews/PnPTab.vue:63]: In general this really should be sent by the backend. We should make an issue to fix this but not in this PR.

Comment (gerth2) [reply photon-client/src/views/PipelineViews/PnPTab.vue:63]: https://github.com/PhotonVision/photonvision/issues/182 in

**Review by mcm001 — APPROVED** [review]

**Review by mcm001 — DISMISSED** [review]

**Comment by gerth2** [comment]
The not-all-that-needed validation for swapping from `floor` to double`for`:
image

**Review by mcm001 — APPROVED** [review]

File: photon-server/src/main/java/org/photonvision/vision/pipeline/Calibrate3dPipeline.java [hunk 2]
```diff
@@ -146,7 +146,7 @@ protected CVPipelineResult process(Frame frame, Calibration3dPipelineSettings se
                         MathUtils.nanosToMillis(sumPipeNanosElapsed),
                         fps,
                         Collections.emptyList(),
-                        new Frame(new CVMat(inputColorMat), frame.frameStaticProperties));
+                        new Frame(new CVMat(outputColorMat), frame.frameStaticProperties));
```
Comment (gerth2) [inline photon-server/src/main/java/org/photonvision/vision/pipeline/Calibrate3dPipeline.java:149]: Note - this is going to potentially conflict functionally with #185 . I'd changed this to assist with debugging in the unit sim and at runtime.

Depending on the merge order, some code may need to be tweaked. The underlying key - there are two `mat`'s in play here. Whichever is _not_ getting returned is the one that needs `release()` called on it.

May also be impacted by some of the discord discussion about architecture and using thresholded vs color frames from picam accelerated sources.

Comment (gerth2) [reply photon-server/src/main/java/org/photonvision/vision/pipeline/Calibrate3dPipeline.java:149]: Redid 185 so this should no longer be a concer.

**Review by Bankst — APPROVED** [review]

**Pull request merged**